src/targets/gpu/CMakeLists.txt

```diff
@@ -237,6 +238,19 @@ else()
     endif()
 endif()
 
+if(NOT WIN32)
+    include(MigxLocalTarget)
+    rocm_local_targets_migx(MIGXGPU_TARGET)
+    message(STATUS "Compiling for gfx arch: ${MIGXGPU_TARGET}")
+
+    foreach(agent IN LISTS MIGXGPU_TARGET)
+        list(APPEND offload_arch_flags "--offload-arch=${agent}")
+    endforeach()
+
+    target_compile_options(migraphx_gpu PRIVATE ${offload_arch_flags})
```

Comment:
`migraphx_gpu` does not have device code.

Reply:
removed.